packages/ariakit/src/tooltip/__examples__/tooltip/test.tsx

```diff
@@ -0,0 +1,20 @@
+import { getByRole, hover, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render tooltip", () => {
+  render(<Example />);
+  expect(getByRole("button")).toMatchInlineSnapshot(`
+    <button
+      aria-labelledby="r:0"
+      tabindex="0"
+    >
+      Hover on me!
+    </button>
+  `);
+});
+
+test("hover tooltip", async () => {
+  render(<Example />);
+  await hover(getByRole("button"));
+  expect(getByRole("tooltip")).toBeVisible();
```

Comment:
Nice 👍 Let's also check if it's not visible before hovering over the button/after leaving it:

```suggestion
  const { baseElement } = render(<Example />);
  expect(getByRole("tooltip", { hidden: true })).not.toBeVisible();
  await hover(getByRole("button"));
  expect(getByRole("tooltip")).toBeVisible();
  await hover(baseElement);
  expect(getByRole("tooltip", { hidden: true })).not.toBeVisible();
```